rules/opentitan/fpga.bzl

```diff
@@ -150,15 +166,36 @@ def _test_dispatch(ctx, exec_env, firmware):
     args = " ".join(args).format(**param)
     args = ctx.expand_location(args, data_labels)
 
-    # Construct the test script
-    script = ctx.actions.declare_file(ctx.attr.name + ".bash")
+    # Construct the opentitantool test setup commands
+    test_setup_cmd = ['--exec="transport init"']
+    if _get_bool(param, "testopt_clear_before_test"):
+        test_setup_cmd.append('--exec="fpga clear-bitstream"')
+    if "bitstream" in param:
+        test_setup_cmd.append('--exec="fpga load-bitstream {bitstream}"')
+    if _get_bool(param, "testopt_bootstrap") and "firmware" in param:
+        test_setup_cmd.append('--exec="bootstrap --leave-in-reset --clear-uart=true {firmware}"')
+    test_setup_cmd = "\n".join(test_setup_cmd)
+    test_setup_cmd = test_setup_cmd.format(**param)
```

Comment:
Most of the custom test-harness tests manage their own bitstream and bootstrapping in the harness.  How does this step interact with those tests? (ie: can we set flags appropriately to end up constructing an empty `test_setup_cmd` so that those tests continue to operate normally)